File: E1/Task3/plot_powerspectrum.py
# plot the powerspectrum
# Created by Martin Gren 2014-10-25.

# imports
import matplotlib.pylab as plt
import numpy as np

# input file
filename = 'powerspectrum.dat'

# import data
data = np.loadtxt(filename)

# initial size of plot window
plt.figure(figsize=(8,6))

# plot
plt.plot(data[:,0], data[:,1],'-')

# labels
plt.xlabel('Frequency / [arb. unit]', fontsize=20)
plt.ylabel('Power spectrum / [arb. unit]', fontsize=20)

# set tick fontsize
plt.yticks(fontsize=12)
plt.xticks(fontsize=12)

# display the plot

plt.savefig('powerspectrum.pdf')
plt.show()

File: E1/Task3/plot_powerspectrum.m
% plot the powerspectrum
% Created by Martin Gren 2014-10-25.
close; % close figs
% load the data file
data = importdata('powerspectrum258.dat');

%plot
figure;
plot(data(:,1),data(:,2),'-');

% labels
xlabel('Frequency','Interpreter','LaTeX');
ylabel('Powers pectrum','Interpreter','LaTeX');
title('f = 2, \psi = 0, n = 258');


